packages/node_modules/@ciscospark/widget-demo/src/components/demo-widget/index.js

```diff
@@ -393,25 +407,52 @@ class DemoWidget extends Component {
                 value="Enable Space List Filter"
                 disabled={this.state.recentsRunning}
               />
-              <div>
-                <Button
-                  ariaLabel="Open Recents Widget"
-                  color="blue"
-                  disabled={!loadRecentsButtonEnabled}
-                  onClick={this.handleOpenRecentsWidget}
-                >
-                  Open Recents Widget
-                </Button>
-                <Button
-                  ariaLabel="Remove Recents Widget"
-                  color="blue"
-                  disabled={!this.state.recentsRunning}
-                  id="removeRecentsWidgetButton"
-                  onClick={this.handleRecentsRemove}
-                >
-                  Remove Recents Widget
-                </Button>
-              </div>
+              <h3>Space Type Filter</h3>
+              <RadioGroup
+                ariaLabel="Space Type Filter"
+                name="spaceTypeFilter"
+                onChange={(e) => this.handleSpaceTypeFilterChange(e)}
+                values={[SPACE_TYPE_ALL]}
```

Comment:
this should be `values={[this.state.spaceTypeFilter]}` so it properly maps to the state value.